src/Apps/Artwork/Components/ArtworkDetails/ArtworkDetailsConditionInfo.tsx

```diff
@@ -0,0 +1,92 @@
+import { Sans } from "@artsy/palette"
+import { Checkmark } from "Assets/Checkmark"
+import colors from "Assets/Colors"
+import React from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+import { Box } from "Styleguide/Elements/Box"
+
+import { ArtworkDetailsConditionInfo_artwork } from "__generated__/ArtworkDetailsConditionInfo_artwork.graphql"
+
+export interface ArtworkDetailsConditionInfoProps {
+  artwork: ArtworkDetailsConditionInfo_artwork
+}
+
+export class ArtworkDetailsConditionInfo extends React.Component<
+  ArtworkDetailsConditionInfoProps
+> {
+  renderRow(label: string, details: string) {
+    if ((!label || label.length < 1) && (!details || details.length < 1)) {
```

Comment:
I think empty strings would be not truthy, so you can probably leave off the `.length` check.